src/users/containers/user-details/user-details.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import * as fromStore from '../../store';
+import * as fromRoot from '../../../app/store';
+import { Observable, Subscription, combineLatest } from 'rxjs';
+import { ActivatedRoute } from '@angular/router';
+import { map } from 'rxjs/internal/operators';
+
+@Component({
+  selector: 'app-prd-user-details-component',
+  templateUrl: './user-details.component.html',
+  styleUrls: ['./user-details.component.scss']
+})
+export class UserDetailsComponent implements OnInit {
+
+  user$: Observable<any>;
+  isLoading$: Observable<boolean>;
+  user: any;
+
+  userSubscription: Subscription;
```

Comment:
if you're not doing anything special with the subscriptions, you could combine these into a single subscription using e.g. subscription.add(), then only need to unsubscribe that single subscription (just a bit cleaner)